extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -496,5 +482,59 @@ private boolean isEqual(State state1, State state2) {
         }
         return colorModeIsEqual && effectIsEqual;
     }
+    
+    private boolean isReachable() {
+        try {
+            // note that InetAddress.isReachable is unreliable, see
+            // http://stackoverflow.com/questions/9922543/why-does-inetaddress-isreachable-return-false-when-i-can-ping-the-ip-address
+            // That's why we do an HTTP access instead
+
+            if (bridge != null) {
+                // If there is no connection, this line will fail
+                bridge.authenticate("invalid");
+            }
+        } catch (IOException e) {
+            return false;
+        } catch (ApiException e) {
+            if (e.getMessage().contains("SocketTimeout") || e.getMessage().contains("ConnectException")
+                    || e.getMessage().contains("NoRouteToHostException")) {
+                return false;
+            } else {
+                // this seems to be only an authentication issue
+                return true;
+            }
+        }
+        return true;
+    }
+
+    @Override
+    public Collection<ConfigStatusMessage> getConfigStatus() {
+        // The bridge IP address to be used for checks
+        final String bridgeIpAddress = (String) getThing().getConfiguration().get(HOST);
+
+        // Check whether an IP address is provided
+        if (bridgeIpAddress == null) {
+
+            return ImmutableList.of(ConfigStatusMessage.Builder.error(HOST)
+                    .withMessageKey(HueConfigStatusMessage.IP_ADDRESS_MISSING.getMessageKey()).build());
+        }
+
+        // Check if the bridge on the given IP address is reachable
+        if (!isReachable()) {
+
+            return ImmutableList.of(ConfigStatusMessage.Builder.error(HOST)
```

Comment:
yes... correct... unreachable hue bridges are already handled by the thing status... would you like to get a config status message for an IP address that does not belong to a hue bridge?